server/plugins/ansible/impl/src/main/java/com/walmartlabs/concord/server/plugins/ansible/PlayInfoProcessor.java

```diff
@@ -83,6 +83,10 @@ public void process(DSLContext tx, List<Event> events) {
                     continue;
                 }
 
+                if (playName.length() > 1024) {
```

Comment:
we can use something like:
```
ANSIBLE_PLAY_STATS.PLAY_NAME.getDataType().length()
```
instead of 1024 constant

Reply:
That's a much better idea.